src/rockstor/fs/btrfs.py

```diff
@@ -934,24 +934,57 @@ def scrub_start(pool, force=False):
 
 
 def scrub_status(pool):
+    """
+    Returns the raw statistics per-device (-R option) of the ongoing or last
+    known btrfs scrub. Works by parsing the output of the following command:
+    btrfs scrub status -R <mount-point>
+    :param pool: pool object
+    :return: dictionary indexed via 'status' and if a finished or halted, or
+    cancelld scrub is indicated then the duration of that scrub is added as
+    value to added index 'duration'. In all 'status' cases bar 'unknown',
+    data_bytes_scrubbed is passed as value to index 'kb_scrubbed' and all
+    other -R invoked details are returned as key value pairs.
+    """
     stats = {'status': 'unknown', }
     mnt_pt = mount_root(pool)
     out, err, rc = run_command([BTRFS, 'scrub', 'status', '-R', mnt_pt])
-    if (len(out) > 1):
-        if (re.search('running', out[1]) is not None):
+    if err != [''] and len(err) > 0:
```

Comment:
To me this line reads as `if len(err) != 0 and len(err) > 0`
can't this just be `if len(err) > 0:`?